api/admin/opds.py

```diff
@@ -31,14 +33,20 @@ def annotate_work_entry(self, work, active_license_pool, edition, identifier, fe
             if measurement.data_source.name == DataSource.LIBRARY_STAFF and measurement.is_most_recent:
                 entry.append(self.rating_tag(measurement.quantity_measured, measurement.value))
 
-        feed.add_link_to_entry(
-            entry,
-            rel="http://librarysimplified.org/terms/rel/refresh",
-            href=self.url_for(
-                "refresh",
-                identifier_type=identifier.type,
-                identifier=identifier.identifier, _external=True)
-        )
+        try:
+            MetadataWranglerCollectionRegistrar(work.license_pools[0].collection)
```

Comment:
Do you not have to assign this to any variable?

Reply:
I don't actually need it - I only need to make sure it doesn't throw an exception. There's code in the controller that actually uses this, and I don't want to show the button if that code would just fail.

Reply:
Makes sense but looks weird to just have this floating around like this.

Reply:
Yeah it does. Do you have a better idea?

Reply:
No :(